src/market/factory.test.ts

```diff
@@ -0,0 +1,55 @@
+import { DemandFactory } from "./factory";
+import { anything, capture, imock, instance, mock, when } from "@johanblumenberg/ts-mockito";
+import { Package } from "../package";
+import { Allocation } from "../payment";
+import { YagnaApi } from "../utils";
+import { RequestorApi as MarketRequestorApi } from "ya-ts-client/dist/ya-market/src/api/requestor-api";
+
+describe("Demand Factory", () => {
+  describe("mid-agreement payments support", () => {
+    describe("default behaviour", () => {
+      test("it configures mid-agreement payments by default", async () => {
```

Comment:
We are using it() everywhere, not test(). I think keeping only one of those would be better for consistency.